app/components/list/list-header.tsx

```diff
@@ -37,8 +34,7 @@ export const ListHeader = ({
 }: ListHeaderProps) => {
   const { modeIsAdvanced } = useAssetIndexMode();
   const freezeColumn = useAssetIndexFreezeColumn();
-  const { originalHeaderRef, isSticky, stickyHeaderRef, coords } =
-    useStickyHeaderPortal();
+  const { originalHeaderRef } = useStickyHeaderPortal();
```

Comment:
We dont need this anymore, however we still need a ref, let me explain.
So I want to add `shadow` to the thead, but it should only be added if the table scroll is not on the top.
So you can use the thead ref to find the parent that is scrollable and check its scroll position and if its not the top, add the shadow.

Reply:
Yes, I got it.